pkg/options/validate.go

```diff
@@ -74,6 +88,29 @@ func LoadReleaseOptionsFromFile(fs billy.Filesystem, path string) (ReleaseOption
 	return releaseOptions, yaml.Unmarshal(releaseOptionsBytes, &releaseOptions)
 }
 
+// SortBySemver sorts the version strings in release options according to semver constraints
+func (r ReleaseOptions) SortBySemver() {
+	for chartName, versions := range r {
+		slices.SortFunc(versions, CompareVersions)
+		r[chartName] = versions
+	}
+}
```

Comment:
Where does this get used?

Reply:
Oh I originally had in mind it would be called before saving the file. But then I forgot to actually _make_ the call, will update.